Tools/refcounts/.ruff.toml

```diff
@@ -0,0 +1,34 @@
+target-version = "py312"
+line-length = 79
+fix = true
+
+[format]
+quote-style = "single"
+
+[lint]
+select = [
+    "ALL",
```

Comment:
I wouldn't really recommend selecting `ALL`, and certainly not in CPython. It can make upgrading to the latest version of Ruff quite hard if there's a new minor release, as we generally stabilise lots of new rules in each minor release. We also have some rules in Ruff that I'd say are somewhat... questionable, or at least _very_ opinionated :-)

I'd probably just select a few categories that you find useful

Reply:
Oh yes. I think I'll go with those that are already used elsewhere.